src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,244 @@
+import DataGrid, { BaseColumn } from "core/components/DataGrid";
+import ChevronLinkColumn from "core/components/DataGrid/ChevronLinkColumn";
+import DateColumn from "core/components/DataGrid/DateColumn";
+import Page from "core/components/Layout/Page";
+import { PageContent } from "core/components/Layout/PageContent";
+import { createGetServerSideProps } from "core/helpers/page";
+import {
+  DashboardPageDocument,
+  useDashboardPageQuery,
+} from "dashboards/graphql/queries.generated";
+import { useTranslation } from "next-i18next";
+import { BookmarkSquareIcon, BookOpenIcon } from "@heroicons/react/24/outline";
+import DescriptionList, {
+  DescriptionListDisplayMode,
+} from "core/components/DescriptionList";
+import Link from "next/link";
+import { ActivityStatus } from "graphql-types";
+import Badge from "core/components/Badge";
+import { useMemo } from "react";
+import clsx from "clsx";
+import { capitalize } from "lodash";
+
+const ActivityStatusBadge = ({ status }: { status: ActivityStatus }) => {
+  const { t } = useTranslation();
+  let className = useMemo(() => {
+    switch (status) {
+      case ActivityStatus.Error:
+        return "bg-red-100 text-red-500";
+      case ActivityStatus.Pending:
+        return "bg-gray-100 text-gray-600";
+      case ActivityStatus.Running:
+        return "bg-sky-100 text-sky-600";
+      case ActivityStatus.Success:
+        return "bg-emerald-50 text-emerald-500";
+    }
+  }, [status]);
+  return (
+    <Badge className={clsx(className, "flex items-center")}>
+      {t(capitalize(status))}
+    </Badge>
+  );
+};
+
+const dummy = [
+  {
+    description: "All datasources are up to date!",
+    occurredAt: "2022-11-09T00:00:39.651Z",
+    url: "/catalog/",
+    status: "ERROR",
+  },
+  {
+    description: "Quentin Did that",
+    occurredAt: "2022-11-09T00:00:39.651Z",
+    url: "/catalog/",
+    status: "ERROR",
+  },
+];
+const DashboardPage = () => {
+  const { t } = useTranslation();
+  const { data } = useDashboardPageQuery();
+
+  return (
+    <Page title={t("Dashboard")}>
+      <PageContent>
+        <Breadcrumbs className="my-8 px-2">
+          <Breadcrumbs.Part href="/pipelines">
+            {t("Dashboard")}
+          </Breadcrumbs.Part>
+        </Breadcrumbs>
+        <div className="my-12">
+          <div className="flex items-center justify-between">
+            <h3 className="ml-2 mt-2 text-lg leading-6 text-gray-900">
+              {t("Overview")}
+            </h3>
+          </div>
+          <div className="mt-4">
+            <div className="grid grid-cols-1 gap-5 sm:grid-cols-2 lg:grid-cols-3">
+              <Block>
+                <div className="p-5">
+                  <div className="flex items-center">
+                    <div className="flex-shrink-0">
+                      <BookmarkSquareIcon className="text-cool-gray-400 h-6 w-6" />
+                    </div>
+                    <div className="ml-5 w-0 flex-1">
+                      <DescriptionList
```

Comment:
Better to not bloat the dom with all the nodes of DescriptionList. You can just replace this by  a `<div />` and a few classes

Reply:
Okay.